elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -459,17 +459,17 @@ elytron.simple-regex-realm-mapper=Definition of a simple RealmMapper that attemp
 elytron.simple-regex-realm-mapper.add=Add operation for the RealmMapper
 elytron.simple-regex-realm-mapper.remove=Remove operation for the RealmMapper
 #Attributes
-elytron.simple-regex-realm-mapper.pattern=The regular expression which must contain at least one capture group to extract the realm from the name.
+elytron.simple-regex-realm-mapper.pattern=The regular expression which must contain at least one capture group to extract the realm from the name. In case when more capture groups are obtained in pattern then first capture group is used.
```

Comment:
I'd say: If the regular expression matches more than one capture group, the first capture group is used.

(Is the ordering of the groups explicit anywhere, so that 'the first' is known?)